README.md

```diff
@@ -14,7 +14,7 @@ Please cite at least https://arxiv.org/abs/2305.19064 if you use this emulator i
 - Create a new conda environment. It is usually better to follow python version one or two behind. In January 2024, the latest is 3.12, so we recommend 3.11.
 
 ```
-conda create -n lace -c conda-forge python=3.11 camb fdasrsf pip=24.0
```

Comment:
The installation may work for you without the packages you deleted, but this is not the case in old clusters. We checked that Nils needed these in order for the installation to work.